curations/nuget/nuget/-/Microsoft.Windows.SDK.Win32Docs

```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: Microsoft.Windows.SDK.Win32Docs
+  provider: nuget
+  type: nuget
+revisions:
+  0.1.42-alpha:
+    licensed:
+      declared: OTHER
```

Comment:
This needs be updated to `LicenseRef-scancode-ms-windows-sdk-win10-net-6`. Also, the file is missing the `.yaml` extension.